tests/unit/FilingHistoryList.spec.ts

```diff
@@ -393,6 +394,62 @@ describe('Filing History List', () => {
     wrapper.destroy()
   })
 
+  it('displays default title if numbered company incorporation application', async () => {
```

Comment:
Is there already a test for a named company incorp app? If not, please add one. Thanks!

Reply:
the other test for this implicitly tests a named company 'displays a Filing Pending incorporation application filing and documents'
(I added an nr number for this) if this is not enough though I can add another test case

Reply:
Thanks, it's fine as is.